src/pages/docs/test/index.mdx

```diff
@@ -0,0 +1,48 @@
+---
```

Comment:
As you've already said in the description, this is gonna be removed before merge as it's a test page. Makes sense to preserve this stuff in the PR description though (bookended by ``` so it maintains formatting) so that we have a record we can look back on